backend/src/core/services/template-client.class.ts

```diff
@@ -115,6 +115,7 @@ export default class TemplateClient {
     } catch (err) {
       logger.error({ message: `${err.stack}` })
       if (err.message.includes('unclosed tag')) throw new TemplateError('There are unclosed curly brackets in the template')
+      if (err.message.includes('unclosed string')) throw new TemplateError(`Invalid character found in keyword. Only alphanumeric characters allowed.`)
```

Comment:
Maybe something like "Check that your keywords only contain letters and numbers. Keywords like {{ Person's Name }} are not allowed, but {{ PersonsName }} will be allowed."